.github/workflows/update-versions.yaml

```
@@ -37,14 +37,15 @@ jobs:
         env:
           GH_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OCP_IGNORED_VERSIONS: '["4.11","4.13"]'
+          OCP_IGNORED_VERSIONS_REGEX: "4.[0-9]|4.1[0-1,3]"
           VERSION_FILE_PATH: "workflows/versions.json"
-          TEST_TO_TRIGGER_FILE_PATH: "workflows/generatble-files/tests_to_trigger.txt"
+          TEST_TO_TRIGGER_FILE_PATH: "workflows/generated-files/tests_to_trigger.txt"
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v7
         with:
-          title: "[Automatic] Update versions"
+          title: "[Automatic] Update versions ${{ github.workflow_job.created_at }}"
```

actionlint (1.7.4)

property "workflow_job" is not defined in object type {action: string; action_path: string; action_ref: string; action_repository: string; action_status: string; actor: string; actor_id: string; api_url: string; base_ref: string; env: string; event: object; event_name: string; event_path: string; graphql_url: string; head_ref: string; job: string; job_workflow_sha: string; path: string; ref: string; ref_name: string; ref_protected: string; ref_type: string; repository: string; repository_id: string; repository_owner: string; repository_owner_id: string; repositoryurl: string; retention_days: number; run_attempt: string; run_id: string; run_number: string; secret_source: string; server_url: string; sha: string; token: string; triggering_actor: string; workflow: string; workflow_ref: string; workflow_sha: string; workspace: string}

(expression)